lib/src/components/NxSmallTag/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { LabelHTMLAttributes, ReactNode } from 'react';
+import * as PropTypes from 'prop-types';
+
+export const NX_SMALL_TAG_COLORS = ['blue', 'green', 'indigo', 'orange', 'pink',
+  'purple', 'red', 'teal', 'turquoise'] as const;
+export type NX_SMALL_TAG_COLOR_TYPE = (typeof NX_SMALL_TAG_COLORS)[number];
+
+export interface Props extends LabelHTMLAttributes<HTMLLabelElement> {
+  children: ReactNode;
+  smallTagColor?: NX_SMALL_TAG_COLOR_TYPE | null;
```

Comment:
This doesn't seem to match the interface in the docs and actual component code.  Isn't it just `color`?

Reply:
Yes it should be `color`, updated here 0fe003abf2cbaef0bb1d32876e5d23c78f26c39c